panopticum/templates/layout/_sidebar.html

```diff
@@ -9,13 +10,22 @@
 
   <!-- menu profile quick info -->
   <div class="profile clearfix">
+  {% if user.is_authenticated %}
     <div class="profile_pic">
-      <img src="/static/images/img.jpg" alt="..." class="img-circle profile_img">
+      <img src="{% if user.photo %}{{user.photo_url}}{% else %}{% static 'images/default_avatar.png' %}{% endif %}"
```

Comment:
I agree at vue.js side and SPA. But we will throw that code to trash bin when we will switch to SPA and vue.js, because It will moved to JS and vue.js side.

Reply:
vue.js is already there, is there anything wrong with it?